extensions/ql-vscode/src/query-history/store/query-history-store.ts

```diff
@@ -14,7 +14,7 @@ import { QueryEvaluationInfo } from "../../run-queries-shared";
 import { QueryResultType } from "../../pure/legacy-messages";
 import { redactableError } from "../../pure/errors";
 
-export async function deserializeQueryHistory(
+export async function readFromQueryHistoryFile(
```

Comment:
Should we keep the naming between these two names consistent by naming it `readQueryHistoryFromFile`?